functions/slack/SlackKnowledgeGraphSearch/SlackRequestVerifier.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace SlackKnowledgeGraphSearch
+{
+    // [START functions_verify_webhook]
```

Comment:
Include the whole class, with usings, etc...

Reply:
Done.